### PR TITLE
revise Debugging Getting Started doc

### PR DESCRIPTION
**Pull request by Trott**
Refs: https://github.com/nodejs/nodejs.org/issues/1968

/cc @Floriferous

### CODE REVIEW

**Review by lpinca — APPROVED** [review]

**Comment by fhemberger** [comment]
Also appears in

```
fa/docs/guides/getting-started-guide.md
ja/docs/guides/getting-started-guide.md
ko/docs/guides/getting-started-guide.md
zh-cn/docs/guides/getting-started-guide.md
```

**Comment by ghost** [comment]
@nodejs/localization：Plz check your own localizations and do translations at **[Trott:debugging-start](https://github.com/Trott/new.nodejs.org/tree/debugging-start/locale)**

File: locale/zh-cn/docs/guides/debugging-getting-started.md [hunk 1]
```diff
@@ -9,27 +9,11 @@ layout: docs.hbs
...
-  "id": "0f2c936f-b1cd-4ac9-aab3-f63b0f33d55e",
-  "title": "node",
-  "type": "node",
-  "url": "file://",
-  "webSocketDebuggerUrl": "ws://127.0.0.1:9229/0f2c936f-b1cd-4ac9-aab3-f63b0f33d55e"
-}
-```
+当使用 `--inspect` 开关符启动检查器时，一个 Node.js 进程开始侦听调试客户端。默认情况下侦听 127.0.0.1:9229 的域名和端口号；每个进程都有一个唯一的 [UUID][] 标示符。
+
+检查器的客户端必须知晓并制定连接的域名地址、端口号以及 UUID。一个典型的 URL 看上去如：`ws://127.0.0.1:9229/0f2c936f-b1cd-4ac9-aab3-f63b0f33d55e`。
```
Comment (ZYSzys) [inline locale/zh-cn/docs/guides/debugging-getting-started.md]: Nit:
```suggestion
检查器的客户端必须知晓并指定连接的域名地址、端口号以及 UUID。一个完整的 URL 看上去如：`ws://127.0.0.1:9229/0f2c936f-b1cd-4ac9-aab3-f63b0f33d55e`。
```

File: locale/zh-cn/docs/guides/debugging-getting-started.md [hunk 2]
```diff
@@ -9,27 +9,11 @@ layout: docs.hbs
...
 
-Node.js 进程如果 *不使用* `--inspect` 开关启动，也可以由指令启动，通过用 `SIGUSR1` 发出信号侦听调试消息（在Linux上 和 OSX）。在 Node 7上，激活了传统的调试器 API；在 Node 8 和以后它将激活监视器 API。
+如果收到了 `SIGUSR1` 信号 (`SIGUSR1` 在 Windows 下不可用)，Node.js 同样会开始侦听调试信息；在 Node.js 7 以及先前的版本中，这将激活遗留版本的调试 API（legacy Debugger API）；在 Nodejs 8 和后续版本中，将激活检查器 API（Inspector API）。
```
Comment (ZYSzys) [inline locale/zh-cn/docs/guides/debugging-getting-started.md]: Nit:
```suggestion
如果收到了 `SIGUSR1` 信号 (`SIGUSR1` 在 Windows 下不可用)，Node.js 同样会开始侦听调试信息；在 Node.js 7 以及先前的版本中，这将激活遗留版本的调试 API（legacy Debugger API）；在 Node.js 8 和后续版本中，将激活检查器 API（Inspector API）。
```

**Review by ZYSzys — APPROVED** [review]
LGTM with some nits.

**Review by vsemozhetbyt — APPROVED** [review]

**Review by willin — APPROVED** [review]

**Comment by fhemberger** [comment]
@nodejs/nodejs-ko Could you please update the Korean translation?

**Comment by yous** [comment]
@outsideris @taggon Please take a look.

**Comment by taggon** [comment]
@nodejs/nodejs-ko I will take a look at this tomorrow.

File: locale/ko/docs/guides/debugging-getting-started.md [hunk 3]
```diff
@@ -67,24 +46,6 @@ it will activate the Inspector API.
 HTTP 요청을 보내서 이 메타데이터를 받을 수 있습니다. 이는 다음과 같은 JSON 객체를 반환합니다.
 인스펙터에 직접 접속하려면 URL로 `webSocketDebuggerUrl` 프로퍼티를 사용하세요.
```
Comment (outsideris) [inline locale/ko/docs/guides/debugging-getting-started.md:47]:
```suggestion
`--inspect` 스위치로 시작하면 Node.js 프로세스가 디버깅 클라이언트에서 수신을 시작하고 
기본 호스트와 포트로 `127.0.0.1:9229`를 사용합니다. 각 프로세스는 고유한 [UUID][]도 할당받습니다.

인스펙터 클라이언트는 호스트 주소, 포트, UUID를 알아야 접속할 수 있습니다. 전체 URL은 
`ws://127.0.0.1:9229/0f2c936f-b1cd-4ac9-aab3-f63b0f33d55e` 같은 형태가 됩니다.

Node.js가 `SIGUSR1` 신호를 받으면 디버깅 메시지도 수신하기 시작합니다.(`SIGUSR1`는 
Windows에서는 사용할 수 없습니다.) Node.js 7 이하 버전에서는 레거시 Debugger API가 활성화되고 
Node.js 8 이상에서는 Inspector API를 활성화합니다. 
```

Comment (outsideris) [reply locale/ko/docs/guides/debugging-getting-started.md:47]: And L35 - L47 should be removed.

**Review by outsideris — CHANGES_REQUESTED** [review]
Please check this @nodejs/nodejs-ko

File: locale/ko/docs/guides/debugging-getting-started.md [hunk 4]
```diff
@@ -67,24 +46,6 @@ it will activate the Inspector API.
...
-  "webSocketDebuggerUrl": "ws://127.0.0.1:9229/0f2c936f-b1cd-4ac9-aab3-f63b0f33d55e"
-}
-```
-
-`--inspect` *없이* 시작된 Node.js 프로세스에도 `SIGUSR1` 신호(Linux와 OS X에서)를 보내서
-디버깅 메시지를 받도록 지시할 수 있습니다. Node 7부터는 레거시 디버거 API를 활성화하고
-Node 8 부터는 인스펙터 API를 활성화할 것입니다.
-
 ---
 <!-- ## Security Implications
```
Comment (yous) [inline locale/ko/docs/guides/debugging-getting-started.md]: Gitpod should be addded below "chrome-remote-interface" at the line 198.

Comment (yous) [reply locale/ko/docs/guides/debugging-getting-started.md]:
``` markdown
#### [Gitpod](https://www.gitpod.io)

* `Debug` 뷰에서 Node.js 디버그 설정을 실행하거나 `F5` 키를 누르세요.
  [자세한 방법은 여기를 참고하세요.](https://medium.com/gitpod/debugging-node-js-applications-in-theia-76c94c76f0a1)
```

**Comment by Trott** [comment]
@outsideris @yous You should be able to push commits directly to my branch if you wish. Feel free to do so if you feel confident about the changes. I'd very much like to get this landed.

**Comment by yous** [comment]
Just pushed updates for Korean that applies reviews.

**Comment by yous** [comment]
@nodejs/nodejs-ko Please check this.

**Comment by Trott** [comment]
Seems like all requests for changes have been addressed? @outsideris

**Review by outsideris — APPROVED** [review]
LGTM

**Comment by fhemberger** [comment]
Thanks everybody!

**Pull request merged**